src/jit/namedintrinsiclist.h

```diff
@@ -9,12 +9,14 @@
 
 enum NamedIntrinsic : unsigned short
 {
-    NI_Illegal                                                  = 0,
-    NI_System_Enum_HasFlag                                      = 1,
-    NI_MathF_Round                                              = 2,
-    NI_Math_Round                                               = 3,
-    NI_System_Collections_Generic_EqualityComparer_get_Default  = 4,
-    NI_System_Buffers_Binary_BinaryPrimitives_ReverseEndianness = 5,
+    NI_Illegal = 0,
```

Comment:
Removed the numbers because I didn't see an explicit need for them.

Having them gone means we can group these alphabetically (which should also generally be more logically grouped as well).